Cargo.toml

```diff
@@ -14,7 +14,7 @@ tensorflow = {version = "0.18.0", features = ["eager"], optional = true }
 plotters = { version = "0.3.0", optional = true }
 tract-onnx = { version = "0.17.7", optional = true }
 anyhow = "1.0.65"
-clap = { version = "4.0.7", features = ["derive"] }
+clap = { version = "4.0.32", features = ["derive"] }
```

Comment:
This change had to be made to resolve an error where some other package needed a different version of a shared dependency.